pkg/fmtutil/fmtutil.go

```diff
@@ -0,0 +1,35 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package fmtutil
+
+import (
+	"text/template"
+
+	"github.com/docker/cli/templates"
+)
+
+// ParseTemplate wraps github.com/docker/cli/templates.Parse() to allow `json` as an alias of `{{json .}}`.
+// ParseTemplate can be removed when https://github.com/docker/cli/pull/3355 gets merged and tagged (Docker 22.XX).
+func ParseTemplate(format string) (*template.Template, error) {
```

Comment:
[`cmd/nerdctl/fmtutil.go:parseTemplate()`](https://github.com/containerd/nerdctl/blob/b71443b2a3abc6288944c72229a2e21a1ae7ec6e/cmd/nerdctl/fmtutil.go#L116-L126) should be now removed

Reply:
aha! Done! Already removed `cmd/nerdctl/fmtutil.go:parseTemplate()`